<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QWidget" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Omega server</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(248, 255, 255)</string>
  </property>
  <widget class="QListWidget" name="lwUsers">
   <property name="geometry">
    <rect>
     <x>419</x>
     <y>70</y>
     <width>161</width>
     <height>141</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background:  rgb(255, 255, 255)</string>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::MultiSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="lwLog">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>361</width>
     <height>181</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 255, 255)</string>
   </property>
   <property name="selectionRectVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="pteMessage">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>441</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(255, 255, 255)</string>
   </property>
   <property name="plainText">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>111</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="topMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QPushButton" name="pbSend">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(225, 225, 225)</string>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="cbToAll">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>To ALL</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pbStartStop">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>250</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(225, 225, 225)</string>
   </property>
   <property name="text">
    <string>Start server</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="chat_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Server logs</string>
   </property>
  </widget>
  <widget class="QLabel" name="usr_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connected users</string>
   </property>
  </widget>
  <widget class="QLabel" name="msg_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Message</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="usr_label_2">
      <property name="text">
       <string> Server IP</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="leHost">
      <property name="styleSheet">
       <string notr="true">background:  rgb(255, 255, 255)</string>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>10</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monotype Corsiva</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>OMEGA SERVER</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
